### DISCUSSION TITLE
Csound audio engine on Cardinal by dbRackCsound Module

### DISCUSSION THREAD

**Discussion by ghost** [DISTRHO/Cardinal#558] [Ideas]
Hi,

modules have been created that allow the use of Csound in VCVRack. Is there any possibility of adding them in future Cardinal releases? Give me your opinion....
The modules are:
https://github.com/docb/dbRackCsound (url)

Csound is:
https://csound.com/ (url)

cheers....

**Comment by falkTX** [comment]
module requests go to https://github.com/DISTRHO/Cardinal/wiki/Possible-modules-to-include

but unlikely we still add more, specially one with such a big dependency

**Reply by ghost** [reply]
Oh, sorry..... my apologize.....

Ok, thanks a lot for Cardinal!

cheers
